x-pack/plugin/core/src/test/java/org/elasticsearch/xpack/core/transform/transforms/TransformHealthIssueTests.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+package org.elasticsearch.xpack.core.transform.transforms;
+
+import org.elasticsearch.common.io.stream.Writeable;
+import org.elasticsearch.test.AbstractXContentSerializingTestCase;
+import org.elasticsearch.xcontent.XContentParser;
+
+import java.io.IOException;
+import java.time.Instant;
+
+public class TransformHealthIssueTests extends AbstractXContentSerializingTestCase<TransformHealthIssue> {
+
+    public static TransformHealthIssue randomTransformHealthIssue() {
+        return new TransformHealthIssue(
+            randomAlphaOfLengthBetween(10, 200),
+            randomBoolean() ? randomAlphaOfLengthBetween(10, 200) : null,
+            randomIntBetween(1, 10),
```

Comment:
If `count` is not compulsory in the parser then this should sometimes be `null`.  (Or alternatively make it a compulsory field in the parser if you prefer.)